kubetest/kind/kind.go

```diff
@@ -249,6 +251,9 @@ func (d *Deployer) Build() error {
 	if d.kindNodeImage != "" {
 		args = append(args, "--image="+d.kindNodeImage)
 	}
+	if kindBaseImage != nil && *kindBaseImage != "" {
```

Comment:
better to have it as a field of `d` (Deployer)

`if d.kindBaseImage != nil {`

https://github.com/kubernetes/test-infra/blob/d90c5aa8a8f75ff007a05ea2324a9fb4c207f4b4/kubetest/kind/kind.go#L86

https://github.com/kubernetes/test-infra/blob/d90c5aa8a8f75ff007a05ea2324a9fb4c207f4b4/kubetest/kind/kind.go#L150

Reply:
@neolit123 we still need the separate variable for the flag, i did try that before ending up with this.

Reply:
Done